kyuubi-common/src/main/scala/org/apache/kyuubi/KyuubiSQLException.scala

```diff
@@ -81,6 +81,10 @@ object KyuubiSQLException {
     }
   }
 
+  def featureNotSupported(): KyuubiSQLException = {
+    KyuubiSQLException("feature not supported", sqlState = "0A000")
+  }
```

Comment:
could you invoke it in the FrontendService for unsupported operation?

Reply:
done